File: huntkingdom2/src/chasse/Entity/Chasse.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package chasse.Entity;

import java.sql.Date;
import java.time.LocalDate;
//import java.util.Date;

/**
 *
 * @author AHMED
 */
public class Chasse {
    private int id;
    private Animal animal;
    //private String saison;
    private String region;
    private int idproduit;
    private Date date_debut;
     private Date date_fin;
     private Type_animal type;
     private int id_user;
     Produitentity name;
     Produitentity produit;
     

    public Chasse(int id, Animal animal, String region, Date date_debut, Date date_fin, Type_animal type, int id_user,  Produitentity produit) {
        this.id = id;
        this.animal = animal;
        this.region = region;
        this.date_debut = date_debut;
        this.date_fin = date_fin;
        this.type = type;
        this.id_user = id_user;
        
        this.produit = produit;
    }

    public Produitentity getProduit() {
        return produit;
    }

    public void setProduit(Produitentity produit) {
        this.produit = produit;
    }

    public Chasse(int id, Animal animal, String region, Date date_debut, Date date_fin, Type_animal type, int id_user, Produitentity name, Produitentity produit) {
        this.id = id;
        this.animal = animal;
        this.region = region;
        this.date_debut = date_debut;
        this.date_fin = date_fin;
        this.type = type;
        this.id_user = id_user;
        this.name = name;
        this.produit = produit;
    }
     


    public Chasse(int id,  Type_animal type ,Animal animal,  String region, Date date_debut, Date date_fin) {
        this.id = id;
        this.animal= animal;
        //this.saison = saison;
        this.region = region;
        this.date_debut = date_debut;
        this.date_fin = date_fin;
        this.type= type;
    }

    public Chasse(Animal animal,Type_animal type, String region, Date date_debut, Date date_fin) {
        this.animal = animal;
        //this.saison = saison;
        this.region = region;
        this.date_debut = date_debut;
        this.date_fin = date_fin;
        this.type= type;
    }

    public Chasse() {
        
    }

    public Chasse(int id, Animal animal, String region, Date date_debut, Date date_fin, Type_animal type, int id_user) {
        this.id = id;
        this.animal = animal;
        this.region = region;
        this.date_debut = date_debut;
        this.date_fin = date_fin;
        this.type = type;
        this.id_user = id_user;
    }

    public Chasse(Animal animal, String region, Date date_debut, Date date_fin, Type_animal type) {
        this.animal = animal;
        this.region = region;
        this.date_debut = date_debut;
        this.date_fin = date_fin;
        this.type = type;
    }

    public int getIdproduit() {
        return idproduit;
    }

    public void setIdproduit(int idproduit) {
        this.idproduit = idproduit;
    }



    public int getId() {
        return id;
    }

    public Animal getAnimal() {
        return animal;  
    }

//    public String getSaison() {
//        return saison;
//    }

    public String getRegion() {
        return region;
    }

    public Date getDate_debut() {
        return date_debut;
    }

    public Date getDate_fin() {
        return date_fin;
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setAnimal(Animal animal) {
        this.animal= animal;
    }

//    public void setSaison(String saison) {
//        this.saison = saison;
//    }

    public void setRegion(String region) {
        this.region = region;
    }

    public void setDate_debut(Date date_debut) {
        this.date_debut = date_debut;
    }

    public void setDate_fin(Date date_fin) {
        this.date_fin = date_fin;
    }

    public Type_animal getType() {

        return type;
    }

    public void setType(Type_animal type) {
        this.type = type;
    }

    public int getId_user() {
        return id_user;
    }

       public void setId_user(int id_user) {
        this.id_user = id_user;
    }
       
    public Chasse( Animal animal, String region, Date date_debut, Date date_fin, Type_animal type,  Produitentity produit) {

        this.animal = animal;
        this.region = region;
        this.date_debut = date_debut;
        this.date_fin = date_fin;
        this.type = type;
       
        
        this.produit = produit;
    }

    @Override
    public String toString() {
        return "Chasse{" + "id=" + id + ", animal=" + animal + ", region=" + region + ", date_debut=" + date_debut + ", date_fin=" + date_fin + ", type=" + type + ", id_user=" + id_user + ", produit=" + produit + '}';
    }




    

  


    

}

File: symfony/project/sympfony/src/HuntkingdomBundle/Entity/Chasse.php
<?php

namespace HuntkingdomBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Chasse
 *
 * @ORM\Table(name="chasse", indexes={@ORM\Index(name="idp", columns={"idp"}), @ORM\Index(name="id_user", columns={"id_user"})})
 * @ORM\Entity
 */
class Chasse
{
    /**
     * @return string
     */
    public function getAnimal()
    {
        return $this->animal;
    }

    /**
     * @param string $animal
     */
    public function setAnimal($animal)
    {
        $this->animal = $animal;
    }

    /**
     * @return string
     */
    public function getRegion()
    {
        return $this->region;
    }

    /**
     * @param string $region
     */
    public function setRegion($region)
    {
        $this->region = $region;
    }

    /**
     * @return \DateTime
     */
    public function getDateDebut()
    {
        return $this->dateDebut;
    }

    /**
     * @param \DateTime $dateDebut
     */
    public function setDateDebut($dateDebut)
    {
        $this->dateDebut = $dateDebut;
    }

    /**
     * @return \DateTime
     */
    public function getDateFin()
    {
        return $this->dateFin;
    }

    /**
     * @param \DateTime $dateFin
     */
    public function setDateFin($dateFin)
    {
        $this->dateFin = $dateFin;
    }

    /**
     * @return string
     */
    public function getType()
    {
        return $this->type;
    }

    /**
     * @param string $type
     */
    public function setType($type)
    {
        $this->type = $type;
    }

    /**
     * @return int
     */
    public function getIdUser()
    {
        return $this->idUser;
    }

    /**
     * @param int $idUser
     */
    public function setIdUser($idUser)
    {
        $this->idUser = $idUser;
    }

    /**
     * @return Product
     */
    public function getIdp()
    {
        return $this->idp;
    }

    /**
     * @param Product $idp
     */
    public function setIdp($idp)
    {
        $this->idp = $idp;
    }
    /**
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param int $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }
    /**
     * @var integer
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="animal", type="string", nullable=false)
     */
    private $animal;

    /**
     * @var string
     *
     * @ORM\Column(name="region", type="string", length=20, nullable=false)
     */
    private $region;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_debut", type="date", nullable=false)
     */
    private $dateDebut;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_fin", type="date", nullable=false)
     */
    private $dateFin;

    /**
     * @var string
     *
     * @ORM\Column(name="type", type="string", nullable=false)
     */
    private $type;

    /**
     * @var integer
     *
     * @ORM\Column(name="id_user", type="integer", nullable=false)
     */
    private $idUser;

    /**
     * @var \HuntkingdomBundle\Entity\Product
     *
     * @ORM\ManyToOne(targetEntity="HuntkingdomBundle\Entity\Product")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="idp", referencedColumnName="ID")
     * })
     */
    private $idp;


}

